src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/AnswerOption.jsx

```diff
@@ -2,57 +2,18 @@ import React, { memo } from 'react';
 import { connect, useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
 import {
-  Col, Collapsible, Icon, IconButton, Form, Row,
+  Collapsible, Icon, IconButton, Form,
 } from '@edx/paragon';
-import { AddComment, Delete } from '@edx/paragon/icons';
-import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Feedback, Delete } from '@edx/paragon/icons';
```

Comment:
Will you change `Delete` to `DeleteOutline`?

Reply:
Going to put this in another pull request when the `Feedback` icon is ready to be changed to `FeedbackOutline`.